internal/conn/conn.go

```diff
@@ -392,13 +391,13 @@ func (c *conn) NewStream(
 ) (_ grpc.ClientStream, err error) {
```

Comment:
it is better don't split the function:
the extracted function has many arguments and many return values. It isn't simple "createStream" it:
1. Create stream.
2. Create and return send mark
3. Create and return traceID

It is difficult to name one common purpose of new function because it make few operations.